VBA/Excel-VBA/articles/excel-performance-and-limit-improvements.md

```diff
@@ -1,247 +1,165 @@
 
 The following sections discuss some of the features that have been introduced in Excel 2016 that you can use to improve performance with large or complex workbooks.
 
 ### Large Address Aware (LAA) Memory improvement for 32-bit Excel
-Although 64-bit Excel has extremely large virtual memory limits, 32-bit Excel has been limited to 2 Gigabytes (GB). And many Excel customers have found it difficult to migrate to 64-bit Excel because they use third-party addins and controls that are not available in 64-bit versions.
-LAA has now been enabled for 32-bit versions of Excel 2013 and Excel 2016, and will minimize out of memory error messages.
-LAA doubles available virtual memory from 2GB to 4GB when using 64-bit Windows, and increases available virtual memory from 2GB to 3GB under 32-bit Windows.
+Although 64-bit Excel has extremely large virtual memory limits, 32-bit Excel has been limited to 2 gigabytes (GB). Many Excel customers have found it difficult to migrate to 64-bit Excel because they use third-party add-ins and controls that are not available in 64-bit versions.
+
+LAA has now been enabled for 32-bit versions of Excel 2013 and Excel 2016, and will minimize out-of-memory error messages. LAA doubles available virtual memory from 2 GB to 4 GB when using 64-bit Windows, and increases available virtual memory from 2 GB to 3 GB under 32-bit Windows.
 
-For more details see [LAA Capability Change for Excel](https://support.microsoft.com/en-ca/help/3160741/large-address-aware-capability-change-for-excel "LAA Capability Change for Excel") 
-To download a tool that shows how much virtual memory is available and how much is being used see [Excel Memory Checking Tool](https://fastexcel.wordpress.com/2016/11/27/excel-memory-checking-tool-using-laa-to-increase-useable-excel-memory/)
+For more information, see [LAA Capability Change for Excel](https://support.microsoft.com/en-ca/help/3160741/large-address-aware-capability-change-for-excel "LAA Capability Change for Excel"). 
 
-### Full Column References
-Previously, workbooks using large numbers of full column references and multiple worksheets, for example `=COUNTIF(Sheet2!A:A,Sheet3!A1)`, might use large amounts of CPU and memory when opened, or rows were deleted. 
-An improvement in Excel 2016 build 16.0.8212.1000 substantially reduces the memory and CPU used in these circumstances.
+To download a tool that shows how much virtual memory is available and how much is being used, see [Excel Memory Checking Tool](https://fastexcel.wordpress.com/2016/11/27/excel-memory-checking-tool-using-laa-to-increase-useable-excel-memory/).
+
+### Full column references
+Previously, workbooks using large numbers of full column references and multiple worksheets (for example `=COUNTIF(Sheet2!A:A,Sheet3!A1)`), might use large amounts of CPU and memory when opened, or rows were deleted. An improvement in Excel 2016 Build 16.0.8212.1000 substantially reduces the memory and CPU used in these circumstances.
```

Comment:
Instead of Previously, say "In earlier versions of Excel...". Also I don't think we need the word "substantially" - I remove almost all adverbs.

We should add "when" between "or" and "rows".

We can also delete "An improvement in...".